src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,19 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    public static final SessionFactory instance = initSessionFactory();
```

Comment:
The field name `instance` is not descriptive enough. Consider renaming it to `sessionFactory` to clearly reflect its purpose.